app/build.gradle

```diff
@@ -283,11 +289,11 @@ dependencies {
     implementation(name: 'hellocharts-library-release', ext: 'aar')
     implementation(name: 'search-preference', ext: 'aar')
     implementation 'org.apache.commons:commons-text:1.3'
-    implementation 'com.google.dagger:dagger:2.11'
+    implementation 'com.google.dagger:dagger:2.25.4'
```

Comment:
Are all these library version changes required to support gradle 5 ?

Reply:
yes, it will not compile with 2.11